File: INF/17/17.cpp
#include <iostream>
#include <string>
#include <fstream>
#include <sstream>
using namespace std;

main() {
	int max = 0, count = 0, prev = 0;
	string s;
	ifstream f;
	f.open("17_2.txt");
	getline(f, s);
	cout << s << '\n';
	prev = stoi(s);
	while(!f.eof()) {
		string str;
		getline(f, str);
		if((prev % 3 == 0 || stoi(str) % 3 == 0) && ((prev + stoi(str)) % 5 == 0)) {
            count++;
            if(prev + stoi(str) > max)
				max = prev + stoi(str);
		}
		prev = stoi(str);
	}
	cout << count << max;
	f.close();
}

File: вариант 15.06/17.py
with open("17 (2).txt") as f:
    mx = 0; mx_s = 0
    count = 0
    lst = []
    for i in f:
        x = int(i)
        lst.append(x)
        if x % 41 == 0 and x > mx:
            mx = x
    for i in range(len(lst) - 1):
        s = lst[i] + lst[i + 1]
        if s < mx:
            count += 1
            if s > mx_s:
                mx_s = s
    print(count, mx_s)
